src/infrastructure/dataSchemas/configDataSchema.ts

```diff
@@ -0,0 +1,34 @@
+import { SemanticVersion } from '@/@types/types'
+
+//NOTE: Add any information about StarryDigitizer configurations which is necassary to reconstruct the application states here.
+export interface configDataSchema {
+  schemaVersion: SemanticVersion
+  figure: {
+    scale: number
+  }
+  axisSets: {
+    id: number
+    pointMode: 0 | 1
```

Comment:
Since it’s a magic number, it would be good to address it later

Reply:
fixed in: https://github.com/t29mato/starry-digitizer/pull/183/commits/5cc683149d68237e3c86ab04ee1c947b874e9bd8